web/client/plugins/Map.jsx

```diff
@@ -182,6 +183,7 @@ let plugins;
 class MapPlugin extends React.Component {
     static propTypes = {
         mapType: PropTypes.string,
+        mapTypeOverride: PropTypes.string,
```

Comment:
Can we implement this behabiour without duplicating mapType ?
mapType is a "global" setting, impacting different plugins. Maybe we should add an option in ContextCreator for it instead of using a Map plugin property.
We already have a UI component (a combobox) to select mapType. We can eventually put that in one of the contextcreator steps.
What do you think?

Reply:
@mbarto good point of view, but it requires not estimated time for this improvement.

Reply:
mmm... yes, can we isolate it in a separate enhancer (with a comment) so we can remove it as far as we found a better solution?
I agree with @mbarto that this plugin, because of it's general purpose, should be as much clear as possible. 
Maybe before connect, so we can use mapType property.



something like 
```javascript
/**
 * If mapType coming as prop ( e.g. from a custom context, it updates the global maptype)
 */
const handleMapType = compose(
    connect(() => ({}), { changeMapType },
    lifecycle({
       componentDidMount()  => { // use this.props.mapType }
   })
)
//...
MapPlugin: compose( 
  handleMapType,
  connect(selector, {
        onFontError: errorLoadingFont,
        onResolutionsChange: setMapResolutions
        onResolutionsChange: setMapResolutions,
        changeMapType
    })(MapPlugin),
```

 What do you think, @mbarto, can this improve a little bit? I know a good solution should be to setup state to load.

In addition I notice that this doesn't stash maptype, so after this the map type will change for all. This is not strictly related to this issue and happens also during navigation, if you manually change the map type from the URL, but anyway we should evaluate and maybe open an issue about it

Reply:
ok for me